client/bindings/pillarEditor.js

```diff
@@ -3,7 +3,16 @@ define(
         'knockout',
         'jquery'
     ], 
-    function(ko, $) { 
+    function(ko, $) {
+        var getPillarModel = function(bindingContext) {
+            if (bindingContext.$parents[2].constructor.name === 'pillarModel') {
+                return bindingContext.$parents[2];
```

Comment:
In the HTML, I am calling this function is called from two different knockout contexts, and this is how I handle that.